src/h2load.cc

```diff
@@ -225,6 +226,12 @@ void rate_period_timeout_w_cb(struct ev_loop *loop, ev_timer *w, int revents) {
   auto nclients_per_second = worker->rate;
   auto conns_remaining = worker->nclients - worker->nconns_made;
   auto nclients = std::min(nclients_per_second, conns_remaining);
+  if (worker->config->warm_up_time > 0) {
+    worker->current_phase = INITIAL_IDLE;
+  }
+  else {
```

Comment:
Nit: Weird indentation here. Try and be consistent with other places in this file.